pkg/devfile/adapters/kubernetes/utils/utils.go

```diff
@@ -156,10 +156,19 @@ func UpdateContainersWithSupervisord(devfileObj devfileParser.DevfileObj, contai
 			// This is done, so supervisord can use it in it's program
 			if !isEnvPresent(container.Env, adaptersCommon.EnvOdoCommandRun) {
 				klog.V(4).Infof("Updating container %v env with run command", container.Name)
+				var setEnvVariable, command string
+				for _, envVar := range runCommand.Exec.Env {
+					setEnvVariable = setEnvVariable + fmt.Sprintf("%v=\"%v\" ", envVar.Name, envVar.Value)
+				}
+				if setEnvVariable == "" {
+					command = runCommand.Exec.CommandLine
+				} else {
+					command = setEnvVariable + " && " + runCommand.Exec.CommandLine
```

Comment:
this leaves a extra space between the last envs and the command
```
ODO_COMMAND_RUN=JAVA_OPTS="some value"  && npm start
```

Reply:
updated.